torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -678,9 +678,15 @@ NvrtcFunction nvrtcCompile(
 #endif
   }
 
-  // Add line info to generated kernels
 #ifndef NDEBUG
```

Comment:
Should we have an env based option as well?

Reply:
I'll add this to `PYTORCH_NVFUSER_DUMP`.